### PR TITLE
chore: new wallet flow

### PR DESCRIPTION
**Pull request by rcresnik**

### CODE REVIEW

File: brd-ios/breadwallet.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -5063,7 +5063,7 @@
 				CODE_SIGN_IDENTITY = "Apple Development";
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 1100;
-				DEVELOPMENT_TEAM = 9YQ85QS3PL;
+				DEVELOPMENT_TEAM = 4MTMK4ZYK4;
```
Comment (kenanmamedoff) [inline brd-ios/breadwallet.xcodeproj/project.pbxproj:5066]: are we sure?

File: brd-ios/breadwallet/src/Strings/da.lproj/Localizable.strings [hunk 2]
```diff
@@ -972,6 +972,8 @@
 
 "Prompts.RateApp.enjoyingBrd" = "Nyder BRD?";
 
+"Prompts.RateApp.enjoyingRockwallet" = "Nyder Rockwallet?";
```
Comment (kenanmamedoff) [inline brd-ios/breadwallet/src/Strings/da.lproj/Localizable.strings:975]: duplicate

File: brd-ios/breadwallet/src/Strings/de.lproj/Localizable.strings [hunk 3]
```diff
@@ -972,6 +972,8 @@
 
 "Prompts.RateApp.enjoyingBrd" = "BRD genießen?";
 
+"Prompts.RateApp.enjoyingRockwallet" = "Rockwallet genießen?";
```
Comment (kenanmamedoff) [inline brd-ios/breadwallet/src/Strings/de.lproj/Localizable.strings:975]: deplicate

File: brd-ios/breadwallet/src/Strings/es.lproj/Localizable.strings [hunk 4]
```diff
@@ -972,6 +972,8 @@
 
 "Prompts.RateApp.enjoyingBrd" = "¿Disfrutas de BRD?";
 
+"Prompts.RateApp.enjoyingRockwallet" = "¿Disfrutas de Rockwallet?";
```
Comment (kenanmamedoff) [inline brd-ios/breadwallet/src/Strings/es.lproj/Localizable.strings:975]: Duplicate

File: brd-ios/breadwallet/src/Strings/fr.lproj/Localizable.strings [hunk 5]
```diff
@@ -972,6 +972,8 @@
 
 "Prompts.RateApp.enjoyingBrd" = "Vous aimez BRD?";
 
+"Prompts.RateApp.enjoyingRockwallet" = "Vous aimez Rockwallet?";
```
Comment (kenanmamedoff) [inline brd-ios/breadwallet/src/Strings/fr.lproj/Localizable.strings:975]: Duplicate

File: brd-ios/breadwallet/src/Strings/it.lproj/Localizable.strings [hunk 6]
```diff
@@ -972,6 +972,8 @@
 
 "Prompts.RateApp.enjoyingBrd" = "Ti piace BRD?";
 
+"Prompts.RateApp.enjoyingRockwallet" = "Ti piace Rockwallet?";
```
Comment (kenanmamedoff) [inline brd-ios/breadwallet/src/Strings/it.lproj/Localizable.strings:975]: Duplicate

File: brd-ios/breadwallet/src/Strings/pt.lproj/Localizable.strings [hunk 7]
```diff
@@ -972,6 +972,8 @@
 
 "Prompts.RateApp.enjoyingBrd" = "Apreciando o BRD?";
 
+"Prompts.RateApp.enjoyingRockwallet" = "Apreciando o Rockwallet?";
```
Comment (kenanmamedoff) [inline brd-ios/breadwallet/src/Strings/pt.lproj/Localizable.strings:975]: Duplicate

File: brd-ios/breadwallet/src/Strings/sv.lproj/Localizable.strings [hunk 8]
```diff
@@ -972,6 +972,8 @@
 
 "Prompts.RateApp.enjoyingBrd" = "Njuter BRD?";
 
+"Prompts.RateApp.enjoyingRockwallet" = "Njuter Rockwallet?";
```
Comment (kenanmamedoff) [inline brd-ios/breadwallet/src/Strings/sv.lproj/Localizable.strings:975]: Duplicate

File: brd-ios/breadwallet/src/ViewControllers/RecoveryKey/RecoveryKeyIntroViewController.swift [hunk 9]
```diff
@@ -254,9 +150,9 @@ class RecoveryKeyIntroViewController: BaseRecoveryKeyViewController {
         didSet {
             if let paging = pagingView {
                 //scrollToItem is broken in the GM build of xcode 12. Check if later versions fix this
-//                paging.scrollToItem(at: IndexPath(item: self.pageIndex, section: 0),
-//                                    at: UICollectionView.ScrollPosition.left,
-//                                    animated: true)
+                //                paging.scrollToItem(at: IndexPath(item: self.pageIndex, section: 0),
+                //                                    at: UICollectionView.ScrollPosition.left,
+                //                                    animated: true)
```
Comment (kenanmamedoff) [inline brd-ios/breadwallet/src/ViewControllers/RecoveryKey/RecoveryKeyIntroViewController.swift:155]: should be removed?

File: brd-ios/breadwallet/src/ViewControllers/RecoveryKey/RecoveryKeyIntroViewController.swift [hunk 10]
```diff
@@ -391,10 +293,9 @@ class RecoveryKeyIntroViewController: BaseRecoveryKeyViewController {
         self.navigationItem.leftBarButtonItem = nil
         self.navigationItem.backBarButtonItem = nil
     }
-        
+    
     private func setUpKeyUseInfoView() {
         keyUseInfoView.text = infoViewText
-        keyUseInfoView.imageName = "ExclamationMarkCircle"
```
Comment (kenanmamedoff) [inline brd-ios/breadwallet/src/ViewControllers/RecoveryKey/RecoveryKeyIntroViewController.swift:397]: should we also delete the asset?

File: brd-ios/breadwallet/src/ViewControllers/RecoveryKey/WriteRecoveryKeyViewController.swift [hunk 11]
```diff
@@ -383,7 +383,6 @@ class WriteRecoveryKeyViewController: BaseRecoveryKeyViewController {
     
     private func updateInfoView() {
         infoView.text = (pageIndex == 0) ? L10n.RecoveryKeyFlow.noScreenshotsOrEmailWarning : L10n.RecoveryKeyFlow.rememberToWriteDownReminder
-        infoView.imageName = (pageIndex == 0) ? "ExclamationMarkCircle" : "Document"
```
Comment (kenanmamedoff) [inline brd-ios/breadwallet/src/ViewControllers/RecoveryKey/WriteRecoveryKeyViewController.swift:386]: should we also delete the assets?

File: brd-ios/breadwallet/src/Constants/Strings.swift [hunk 12]
```diff
@@ -1219,7 +1219,7 @@ internal enum L10n {
     /// Onboarding screen 'Get started' (create new wallet) button
     internal static let getStarted = L10n.tr("Localizable", "Onboarding.getStarted", fallback: "Get started")
     /// Onboarding screen 'Next' button
-    internal static let next = L10n.tr("Localizable", "Onboarding.next", fallback: "Next")
+    internal static let next = L10n.tr("Localizable", "Onboarding.next", fallback: "NEXT")
```
Comment (kenanmamedoff) [inline brd-ios/breadwallet/src/Constants/Strings.swift:1222]: Are we doing these in localization?

File: brd-ios/breadwallet/src/ViewControllers/RecoveryKey/RecoveryKeyIntroViewController.swift [hunk 13]
```diff
@@ -278,6 +174,12 @@ class RecoveryKeyIntroViewController: BaseRecoveryKeyViewController {
                     } else {
                         self.showBackButton()
                     }
+                    
+                    guard self.pages.count == self.pageIndex + 1 else {
+                        self.continueButton.setType(type: .tertiary)
+                        return
+                    }
+                    self.continueButton.setType(type: .secondary)
```
Comment (kenanmamedoff) [inline brd-ios/breadwallet/src/ViewControllers/RecoveryKey/RecoveryKeyIntroViewController.swift:182]: can probably be a one liner

**Review by dangelovska — APPROVED** [review]

**Review by kenanmamedoff — APPROVED** [review]

**Pull request merged**